includes/Blocks/CoreSeparator.php

```diff
@@ -0,0 +1,27 @@
+<?php
```

Comment:
@theodesp not sure why this file is showing, since the same was merged in #113, unless you branched without updating `main` first?